class AuthFailedError(Exception):
    pass

class InvalidResponseError(Exception):
    pass